pipeline-templates/job/build-and-train.yml

```diff
@@ -0,0 +1,50 @@
+parameters:
+  Environment:
+  ServiceConnection:
+
+jobs:
+  - deployment: DeployInfra

```

Comment:
Can the list indenting be removed?  Convention is to align list items in the same column as the parent property